### PR TITLE
Explain what node is broken in assert msg

### PR DESCRIPTION
**Pull request by brabadu**
I got this error for a big graph and it's hard to understand which node is to blame.

### CODE REVIEW

File: hiku/expr/checker.py [hunk 1]
```diff
@@ -132,7 +132,7 @@ def visit_tuple_generic(self, node):
         assert isinstance(sym, Symbol), type(sym)
         args = [self.visit(val) for val in node.values[1:]]
         fn_type = node_type(self.types, sym)
-        assert len(fn_type.__arg_types__) == len(args), 'Wrong arguments count'
+        assert len(fn_type.__arg_types__) == len(args), f'Wrong arguments count at "{sym.name}" node'
```
Comment (vmagamedov) [inline hiku/expr/checker.py]: Seems like `sym.name` is not a node's name, it is a function's name. Could you rephrase this error?

Comment (vmagamedov) [reply hiku/expr/checker.py]: And minimum supported Python version is 3.5, so we can't use f-strings yet

**Comment by brabadu** [comment]
Line is a liitle bit too long now :) Not sure how you would like to format it better though

**Comment by vmagamedov** [comment]
Thanks!
